### PR TITLE
Updates to include latest sdk components available on mojaloop

### PR DESCRIPTION
**Pull request by elnyry**
- Upgrade sdk to @mojaloop/sdk-standard components 7.4.0
- other package updates

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,8 +35,8 @@
   },
```
Comment (rmothilal) [inline package.json]: Please updated the version of the package.json to the next valid version

File: package.json [hunk 2]
```diff
@@ -35,8 +35,8 @@
...
   "homepage": "https://github.com/mojaloop/central-services-error-handling#readme",
   "dependencies": {
-    "@modusbox/mojaloop-sdk-standard-components": "0.0.39",
-    "@mojaloop/central-services-shared": "6.4.1",
+    "@mojaloop/sdk-standard-components": "7.4.0",
+    "@mojaloop/central-services-shared": "7.4.3",
```
Comment (rmothilal) [inline package.json]: version 7.4.4 is released

Comment (elnyry) [reply package.json]: Updated..

File: package.json (hunk 1 above)
Comment (elnyry) [reply package.json]: Its already 7.4.0 and we don't have a release yet in 7.4

**Pull request merged**